doc/en/customize.rst

```diff
@@ -97,6 +97,19 @@ check for ini-files as follows::
 .. _`how to change command line options defaults`:
 .. _`adding default options`:
 
+Change which files are added to sys.path
+-----------------------------------------------
+
+If you place a conftest.py file in the root directory of your project
+(as determined by pytest, see above.) pytest will run tests against 
+the code below that directory by adding it to your sys.path instead of 
```

Comment:
> Either way, the thing I really think should be more clearly stated is that the user is expected to install the python code to be tested in develop mode.

I think this is the heart of the matter; this is good practice and will be useful with any Python project (at least OS that is). So I think removing the sys.path hackery altogether is best in the long term.